sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1196,7 +1196,12 @@ def create_operation(self,
     creator, parameter_type = self._known_urns[transform_proto.spec.urn]
     payload = proto_utils.parse_Bytes(
         transform_proto.spec.payload, parameter_type)
-    return creator(self, transform_id, transform_proto, payload, consumers)
+    operation = creator(self, transform_id, transform_proto, payload, consumers)
+    if common_urns.actuate_pushdown_annotation in transform_proto.annotations:
```

Comment:
This approach requires the worker to have access to the runner API proto (or at least some derivative of it that also contains the pushdown annotations). Do you know if that's the case for Dataflow's legacy Java worker?

cc @apilloud